### PR TITLE
[RAC-5468]on-taskgraph docker image's latest version in DockerHub are not getting updated correctly

### PR DESCRIPTION
**Pull request by HQebupt**
In order to figure out who when where build a docker image and push to the dockerhub.
Related:
- [RackHD/on-tasks/pull/479](https://github.com/RackHD/on-tasks/pull/479)
- [RackHD/on-http/pull/700](https://github.com/RackHD/on-http/pull/700)

**NOTICE**: It will modify the docker file in order to add more information into the docker image which is not expected in production environment.
**After solving the issue, I will discard the changes.**
@changev @panpan0000 @PengTian0 @iceiilin @anhou

### CODE REVIEW

File: build_debug/build_image_info.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+set -x
+
+cd $(dirname $0)
+
+tty=$(who)
+hostname=$(hostname)
+ip_info=$(ifconfig | grep "inet addr")
+os_info=$(lsb_release -a)
+
+echo -e "tty:\n${tty}\n\nhostname:\n${hostname}\n\nip_info:\n${ip_info}\n\nos_info:\n${os_info}" | openssl rsautl -encrypt -pubin -inkey build_image_key.pub > build_image_info.encrypted
```
Comment (anhou) [inline build_debug/build_image_info.sh]: I think 'tty' is useless, and 'who' should be the user, like 'jenkins@vmslave', and also I didn't see 'when'

File: build_debug/build_image_info.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# Purpose: figure out RAC-5468. 
+# If you want to decrypt the file "build_image_info.encrypted", please contact Qiang, Peter, Alan, Leo.
+
+set -x
+
+cur_time=$(date)
+cur_user=$USER
+cur_path=$(pwd)
+
+user_login=$(who)
```
Comment (panpan0000) [inline build_debug/build_image_info.sh]: any diff than cur_user?

File: build_debug/build_image_info.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+hostname=$(hostname)
+ip_info=$(ifconfig | grep "inet addr")
+os_info=$(lsb_release -a)
+
+cd $(dirname $0)
+echo -e "current time:${cur_time}\ncurrent user:${cur_user}\ncurrent path:${cur_path}\n\nuser login:\n${user_login}\n\nhostname:\n${hostname}\n\nip info:\n${ip_info}\n\nos info:\n${os_info}" | openssl rsautl -encrypt -pubin -inkey build_image_key.pub > build_image_info.encrypted 
+
```
Comment (panpan0000) [inline build_debug/build_image_info.sh]: BTW: I'm thinking , can we just dump the whole "ENV" ?

File: build_debug/build_image_info.sh (hunk 2 above)
Comment (HQebupt) [inline build_debug/build_image_info.sh]: "who" displays who is logged on to the system. The output result is more than one user probably.
If Alan and I login on the same machine both , there are two users with the command "who".

File: build_debug/build_image_info.sh (hunk 3 above)
Comment (HQebupt) [inline build_debug/build_image_info.sh]: The whole "ENV" has redundant information, so we can grep the useful info.

**Comment by changev** [comment]
+1 after pasting test results

**Comment by changev** [comment]
docker build run in docker containers 

so all the the env vars of hosts can't be got in docker images~

Please read docker doc and find a way to pass host envs to docker containers.

**Comment by HQebupt** [comment]
The script in docker file is running in the docker environment, not in the host. So the PR could not help to solve the issue.

**Pull request closed without merging**